src/main/java/org/codehaus/mojo/versions/SetPropertyMojo.java

```diff
@@ -63,6 +66,15 @@
     @Parameter( property = "autoLinkItems", defaultValue = "true" )
     private boolean autoLinkItems;
 
+    /**
+     * A property file name containing:
+     *     property1=value1
+     *     ...
+     *     propertyN=valueN
+     */
```

Comment:
Please add the JavaDoc annotation `@Since`, so that users can see since which version the new property is available. I think currently you should set it to 2.9.

Reply:
Done.